fedmsg_meta_fedora_infrastructure/pagure.py

```diff
@@ -284,10 +284,22 @@ def subtitle(self, msg, **config):
             return tmpl.format(user=user, project=project, fields=fields)
         elif 'pagure.project.user.added' in msg['topic']:
             new_user = msg['msg']['new_user']
+            access = msg['msg']['access']
```

Comment:
This is not backward compatible, while fedmsg_meta needs to be able to handle both new and old format of messages